src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.SourceGenerators.Testing/SourceGeneratorTest`1.cs

```diff
@@ -22,6 +20,11 @@ namespace Microsoft.CodeAnalysis.Testing
     public abstract class SourceGeneratorTest<TVerifier> : AnalyzerTest<TVerifier>
         where TVerifier : IVerifier, new()
     {
+        /// <summary>
+        /// Gets a collection of global options that will be awaiable from Context.AnalyzerConfigOptions.GlobalOptions
+        /// </summary>
+        public List<(string key, string value)> GlobalOptions { get; } = new List<(string, string)>();
```

Comment:
This is already provided by TestState.AnalyzerConfigFiles.

Reply:
what about msbuild files? would our approach then be to have the user supply the props/targets files?

Reply:
No, the testing focuses on the compiler component of a build.

Reply:
we support adding nuget packages which is not a compiler component of a build. I don't think API is what we want but we don't think providing a way to test this makes sense?

Reply:
I don't understand the request here. We already have the ability to test global options through an existing API. Why would we need to add a new one?